R/g.report.part5.R

```diff
@@ -1,6 +1,7 @@
 g.report.part5 = function(metadatadir = c(), f0 = c(), f1 = c(), loglocation = c(),
```

Comment:
I am struggling to find the place where you decide on the columns of the segment reports.

Based on looking at the csv report I created from a test file I am wondering whether we need to remove all columns related to the description of the SPT window? Those estimates are only relevant at day or person level and not at segment level. For example, I see that each segment has a different wake up time and sleep onset time, this could be very confusing.

Reply:
In my tests, the sleep onset and wakeup times for the segments in the same day are identical (as expected), see example:
![image](https://github.com/wadpac/GGIR/assets/28734336/4ed7a784-7703-481a-b697-fc28cc8a4a11)

I though about removing these columns from the segment report, but at the same time I thought they do not do any harm there, and they could be helpful to understand why a specific segment has too much or too low SPT or day time. At least it was helpful for me when testing this functionality. What do you think? Should we ignore them in those reports anyway?

Reply:
Thanks for clarifying. Ok, I now see it is correct in my test data as I had not ordered the rows by date.

Yes, maybe this is useful for some users and re-inserting the columns would be a lot harder for the user then to remove the columns. So, let's keep the columns.